get-accessions-from-sbt.py

```diff
@@ -12,13 +12,13 @@ def main():
     p.add_argument('-o', '--output', type=argparse.FileType('wt'))
     args = p.parse_args()
 
-    db = sourmash_lib.load_sbt_index(args.sbt)
+    db = sourmash.load_file_as_index(args.sbt)
 
     for n, leaf in enumerate(db.leaves()):
```

Comment:
```suggestion
    for n, leaf in enumerate(db.signatures()):
```
this should make it work for any sourmash database, not just SBTs. But also for SBTs :)

Oh, and you'd need to change `leaf.data.name` to just `leaf.name`, since `leaf` would now be a `SourmashSignature` object.

Probably better to just rename `leaf` to `ss` or something.

Reply:
you could also grab `db.manifest` and iterate over `rows`; each element in `rows` is a dictionary containing `row['name']` which will give you the info you want here, without actually loading the underlying sketch data.